xformers/csrc/attention/hip_fmha/ck_tiled_fmha_batched_forward.h

```diff
@@ -0,0 +1,233 @@
+/*
```

Comment:
I'm wondering if these files should not rather live in the `ck` repo (similar to CUTLASS examples), and we could just import them from the C++ files in xFormers?
For example we have these kernels which just import a specific CUTLASS example:
https://github.com/facebookresearch/xformers/blob/d93b3fdd6269ec02aa3b0b966b3eef3f18cf0379/xformers/csrc/swiglu/cuda/dual_gemm_silu_identity_mul.cu#L15

Reply:
It is a very reasonable suggestion. However, it will take us some effort. What is your preference, 1) delay and fix it in this upstream or leave it to the next upstream?

Reply:
It's fine to leave it in xFormers for now, we can change that later